.yarnrc.yml

```diff
@@ -1,6 +1,7 @@
 # yaml-language-server: $schema=https://yarnpkg.com/configuration/yarnrc.json
 enableGlobalCache: true
 
+# We rely on a node_modules based setup for serving assets (e.g. nodeModulesScripts).
```

Comment:
👀 was this surprising enough that it warranted calling out? I don't think it's that unusual to avoid Yarn's PnP style, especially for non-greenfield projects.

Reply:
I didn't immediately make the connection tbh 😅